src/main.js

```diff
@@ -242,21 +242,26 @@ function createWindow () {
           },
         },
         {
-          label: 'Changelog',
+          label: 'Report an issue',
           click:  function() {
-            shell.openExternal('https://github.com/krydos/todoist-linux/releases');
+            shell.openExternal('https://github.com/KryDos/todoist-linux/issues/new');
           },
         },
         {
-          label: 'Report an issue',
+          label: 'Changelog',
           click:  function() {
-            shell.openExternal('https://github.com/KryDos/todoist-linux/issues/new');
+            shell.openExternal('https://github.com/krydos/todoist-linux/releases');
```

Comment:
It's interesting how in those two cases you made the exact opposite decision on how to change "krydos" (`krydos` to `KryDos`; and `KryDos` to `krydos`). :-) 

For what it's worth, my browser displays the address bar as `KryDos`.

And just so you know, I don't care if you change it or leave it be. It was just an peculiar observation. :-)

Reply:
Nice catch. IDK how I managed to change the casing, but now they are the same.